src/Documentation/deployment/toc.yml

```diff
@@ -1,14 +1,16 @@
 - name: Running the Application
   href: index.md
+- name: Cluster Management
+  href: cluster_management.md
+- name: Multi-Cluster Support
```

Comment:
No need to move the multi-cluster link up. Let's keep it where it was - at the bottom of the list..